tests/ci/cdk/run-cdk.sh

```diff
@@ -157,12 +155,10 @@ function deploy() {
     "ubuntu-18.04_clang-6x_latest"
```

Comment:
Aren't we missing the Fedora image with Clang in this list?

Reply:
I will address this part when publishing gcc10.

Reply:
Fair and square :)